index.md

```diff
@@ -2,8 +2,26 @@
 # Feel free to add content and custom Front Matter to this file.
 # To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
 
-layout: home
+layout: default
 ---
 
 
-{% google_cse %}
\ No newline at end of file
+{% google_cse %}
+
+
+
```

Comment:
Here we can add a heading "Home" and display the categories as follows.
![Screenshot (376)](https://user-images.githubusercontent.com/48858628/78642835-f9574c00-78d0-11ea-90c4-bc4a66f4294a.png)
That would look more organized and better.